### ISSUE TITLE
question: How can I filter the services to be load? SERVICES env var does not seem to do it now

### ISSUE THREAD

**Issue by mromagnoli** [localstack/localstack#6007]
### Is there an existing issue for this?

- [X] I have searched the existing issues and read the documentation

### Question

Hi everyone, I'm using version `0.14.2` and I'm trying to only enable 2 services but when starting it enables all the services.
I also tried to use the `./localstack/default.env` file to set env vars but it doesn't seem to be read.

This is my config in my `docker-compose.yml` based on [repo's one](https://github.com/localstack/localstack/blob/master/docker-compose.yml)

```
localstack:
    image: localstack/localstack:0.14.2
    container_name: localstack_main
    ports:
      - "127.0.0.1:14510-14559:4510-4559"  # external service port range
      - "127.0.0.1:6645:4566"            # LocalStack Edge Proxy
    environment:
      - SERVICES=dynamodb,s3
      - TMPDIR=/tmp/localstack
      - CONFIG_PROFILE=default    # <--- this should read the host's `.localstack/default.env` AFAIK
      - LAMBDA_REMOTE_DOCKER=false
      - DATA_DIR=${TMPDIR}/data
      - HOST_TMP_FOLDER=${TMPDIR}
    volumes:
      - "./.localstack:/tmp/localstack"
      - "/var/run/docker.sock:/var/run/docker.sock"
    networks:
      - my-net
```
The host `ports` have been changed from the suggested ones to no conflict with other services using them.

`SERVICES` env was removed from the example `docker-compose.yml` file but I don't know what to use now.

Thanks in advance.

### Anything else?

If I query `localhost:6645/health`
All services appear as `available`
```
{
    "features": {
        "initScripts": "initialized"
    },
    "services": {
        "acm": "available",
        "apigateway": "available",
        "cloudformation": "available",
        "cloudwatch": "available",
        "config": "available",
        "dynamodb": "available",
        "dynamodbstreams": "available",
        "ec2": "available",
        "es": "available",
        "events": "available",
        "firehose": "available",
        "iam": "available",
        "kinesis": "available",
        "kms": "available",
        "lambda": "available",
        "logs": "available",
        "opensearch": "available",
        "redshift": "available",
        "resource-groups": "available",
        "resourcegroupstaggingapi": "available",
        "route53": "available",
        "route53resolver": "available",
        "s3": "available",
        "s3control": "available",
        "secretsmanager": "available",
        "ses": "available",
        "sns": "available",
        "sqs": "available",
        "ssm": "available",
        "stepfunctions": "available",
        "sts": "available",
        "support": "available",
        "swf": "available"
    },
    "version": "0.14.2"
}
```

**Comment by dfangl** [comment]
Hi there!
We removed the SERVICES variable from the default docker-compose file to prevent misuse. Since 0.13, we enabled lazy service loading. So while the services are shown as available, they won't actually start up unless you actually use them (or a service you use uses them, for example something pushing logs to cloudwatch will startup the cloudwatch service).

While you can still use the SERVICES variable, it will just disable the not-specified services, which can lead to failures (e.g. disabling logs), but will not impact your startup time in any significant manner.

To sum up: You can still use it to disable services, but it will not give you any advantage in terms of startup time, since they are loaded lazily anyways.

**Comment by mromagnoli** [comment]
Hi @dfangl, thank you very much for your response, now is much clearer.
So, in that case, I shouldn't be surprised by the fact that many ports are open besides the Edge proxy ones. 
For example:
```
$ docker-compose ps
                Name             Command        State        Ports      
----------------------------------------------------------------------
my-localstack        docker-entrypoint.sh             Up      
                                          4510/tcp,
                                          4511/tcp,       
                                          4512/tcp,       
                                          4513/tcp,       
                                          4514/tcp,       
                                          4515/tcp,       
                                          4516/tcp,       
                                          4517/tcp,       
                                          4518/tcp,       
                                          4519/tcp,       
                                          [...],
                                          0.0.0.0:6645->4566/tcp
```

**Comment by dfangl** [comment]
These ports are needed for all resources which are not served over the edge proxy. For example, when you start a instance of mysql in RDS, it will be exposed through this ports. All AWS API actions will be served through 4566 (and possibly 443 in pro).
These ports will only be used when you start a resource which needs this, for example a RDS instance, ElasticSearch/Opensearch with port strategy, ECR etc.
